### DISCUSSION TITLE
web server SSL?

### DISCUSSION THREAD

**Discussion by lukasmrtvy** [postalserver/postal#1477] [Feature suggestions]
Is possible to run web server with SSL? 
fast_server can handle this. I know its best practice to put reverse proxy in front of it, but I am missing a point why web server can not handle SSL too.
Thanks

**Comment by willpower232** [comment]
I don't know the ins and outs but I believe the fast server runs its own SSL as it has to handle domains that are configured in Postal so would be annoying to configure manually. The fast server also only needs to handle small-scale responses so it doesn't have to be as advanced as the web server.

The web interface handles the API requests as well as the UI so its easier to offload the work to nginx.

**Comment by lukasmrtvy** [comment]
It should be easy to edit https://github.com/atech/postal/blob/master/config/puma.rb to allow HTTPS for puma.
Here is info.
https://github.com/puma/puma#controlling-ssl-cipher-suites

Unfortunately I can not help.

**Comment by willpower232** [comment]
Thanks for your research but unless you or someone makes a pull request to add it, I don't think it would happen.

**Comment by lukasmrtvy** [comment]
PR: https://github.com/atech/postal/pull/676
